docs/UPGRADING.rst

```diff
@@ -0,0 +1,20 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python setup.py install
+
+3. Upgrade the database::
+
+     $ haas-admin db upgrade heads
+
+4. If additional extensions have been added to ``haas.cfg``, re-run ``haas db
+   create``, which will create any tables needed by those extensions.
```

Comment:
1. Do sysadmins need to worry about removed extensions?
2. Should this be `haas-admin db create`?
3. Does `upgrade` do this as well?